### DISCUSSION TITLE
dGPU fan control by iGPU temperature

### DISCUSSION THREAD

**Discussion by Lankicu** [seerge/g-helper#1695] [Ideas]
I used FLOW X13 (2021) GV301QC

When the dGPU is turned on but in standby mode, could not update GPU temperature and fan speed control are not performed.
I used more CPU performance power than GPU.
Almost GPU has been idle state. when idle state, G-Helper and windows 10 can not be updated dGPU temperature.
In this state GPU fan is turn off and CPU throttling hard.

So I need option for enable fan control based on iGPU temperature even when dGPU is On.

![dGPU idle](https://github.com/seerge/g-helper/assets/152945266/81e41f07-dd82-41c9-a6bc-34309f3c747f)

**Comment by seerge** [comment]
@Lankicu G-Helper doesn't and can't control fans individually anyhow (same as Armoury).  Fans are controlled by BIOS based on it's own temperature sensors on the motherboard
